index/segment/mem/terms_dict_bench_test.go

```diff
@@ -114,79 +84,35 @@ func benchmarkInsertTrigramTermsDict(docs []doc.Document, b *testing.B) {
 }
 
 func benchmarkMatchTermSimpleTermsDict(docs []doc.Document, b *testing.B) {
-	b.ReportAllocs()
```

Comment:
nit: `ReportAllocs` only affects the output of the test to include allocation information automatically. Perhaps its worth leaving it as a setup step just to be consistent?

Reply:
fair enough